Victor-Ugwueze/MyDiary
# 159310290 Feature get single entry

### What does this PR do ?
This PR refactors entry model to use database connection,

Add testing for get all entries route
Add test for get single entry endpoint
Regenerate heroku api_key to correct Travis CI test failure.

### Description of tasks to be completed ?
Implement the following :
Finish get single entry
Add test for get single entry point,
Refactor entry model and controller to make test pass

### How should this be tested ?

You should have PostgreSQL installed before you get started. Then clone this repo, run npm install to install all dependencies and devDependencis.
Open server/helpers/dbHelper.js change the connection string to match that of your local machine.
Then run `npm test` , and then `npm start-dev`
start POSTMAN
`POST /auth/login` , enter email as `test9@gmail.com`, password `test123`
then access `GET /api/v1/entries/:id` pass the received token, as `x-access-token` on request header

### What are the relevant pivotal tracker stories?
#159310290

### Any background context you wish to provide?
None
